src/pp.js

```diff
@@ -78,6 +78,13 @@ function iterDoc(topDoc, func) {
         for (var i = doc.parts.length - 1; i >= 0; i--) {
           docs.push(doc.parts[i]);
         }
+      } else if (doc.type === "if-break") {
+        if (doc.breakContents !== undefined) {
```

Comment:
Actually, I think I see what this is done. Didn't realize this was in `iterDoc`. I think it makes to do this if we want to encourage people to do `ifBreak("", " ")`: use an empty string instead of null. It's up to you.